lib/SGListView.js

```diff
@@ -41,6 +41,31 @@ const PrivateMethods = {
     }
   },
 
+  makeFrame(cellData, childFrames, sectionID, rowID, horizontal, index) {
```

Comment:
Can you add a description of this function using the javadoc format? What are we considering a `frame` here? Frames makes me thing we're generating a wrapper `<View />` , but it looks like we're returning `{ x, y, width, height, index }`. Would `calculateDimensionsForCell` be more descriptive?

ie.
```
/*
 * This function does...
 */